src/Config.groovy

```diff
@@ -34,6 +34,11 @@ class Config {
       }
     }
 
+    def prjRulesByConvention = new File(appContext.codeFolder, "ruleset.xml")
```

Comment:
Is `prj` an abbreviation for `project`? Is it a common one in Groovy? If not, I'd suggest using the full word instead.